[2025-03-14T19:06:18.605+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T19:06:18.680+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_weather_etl.load scheduled__2025-03-14T19:00:00+00:00 [queued]>
[2025-03-14T19:06:18.683+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_weather_etl.load scheduled__2025-03-14T19:00:00+00:00 [queued]>
[2025-03-14T19:06:18.684+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-14T19:06:18.688+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load> on 2025-03-14 19:00:00+00:00
[2025-03-14T19:06:18.690+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=77) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-14T19:06:18.691+0000] {standard_task_runner.py:72} INFO - Started process 78 to run task
[2025-03-14T19:06:18.691+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'open_weather_etl', 'load', 'scheduled__2025-03-14T19:00:00+00:00', '--job-id', '123', '--raw', '--subdir', 'DAGS_FOLDER/open_weather_dag.py', '--cfg-path', '/tmp/tmp8dqrm3kp']
[2025-03-14T19:06:18.692+0000] {standard_task_runner.py:105} INFO - Job 123: Subtask load
[2025-03-14T19:06:18.717+0000] {task_command.py:467} INFO - Running <TaskInstance: open_weather_etl.load scheduled__2025-03-14T19:00:00+00:00 [running]> on host 762baad180ec
[2025-03-14T19:06:18.746+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='lherrera5034@gmail.com' AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='open_weather_etl' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T19:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-14T19:00:00+00:00'
[2025-03-14T19:06:18.747+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T19:06:18.752+0000] {logging_mixin.py:190} INFO - Data Loaded into Postgres database
[2025-03-14T19:06:18.752+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-14T19:06:18.756+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T19:06:18.756+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=open_weather_etl, task_id=load, run_id=scheduled__2025-03-14T19:00:00+00:00, execution_date=20250314T190000, start_date=20250314T190618, end_date=20250314T190618
[2025-03-14T19:06:18.796+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T19:06:18.803+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-14T19:06:18.804+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
